app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class WebhooksController < ApplicationController
```

Comment:
Its a good point, none of our controllers are in the ShopifyApp namespace but they probably should be. Should probably be a different PR though